CHANGELOG.md

```diff
@@ -27,6 +27,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fixed a `TavusVideoService` issue that was causing audio choppiness.
+
+- `DeepgramTTSService` now uses HTTP requests directly instead of using
+  Deepgram's SDK. Deepgram's SDK was somehow blocking the event loop causing the
+  audio to be choppy.
```

Comment:
Interesting. Is it worth to open an issue for them ?

Reply:
I removed this change. Just used their async version instead, it seems to work in normal case (no Tavus involved).